
string robot_name
std_msgs/Float64 distance_travelled